src/usr/local/www/head.inc

```diff
@@ -461,8 +461,32 @@ if (($pagename === "index.php") && ($numColumns > 2)) {
 	}
 ?>
 		<ul class="context-links">
-<?php
 
+	<?php if ($dashboard_available_widgets_hidden): ?>
+		<li>
+			<a onclick="$('#widget-available').toggle(360);" title="<?=gettext("Show/Hide available widgets panel")?>">
+				<i class="btn-sm btn-breadcrumbbar">Widgets</i>
+			</a>
+		</li>
+	<?php endif?>
+
+	<?php if ($system_logs_filter_form_hidden): ?>
+		<li>
+			<a onclick="$('#filter-form').toggle(360)" title="<?=gettext("Show/Hide log filter panel")?>">
+				<i class="btn-sm btn-breadcrumbbar">Filter</i>
+			</a>
+		</li>
+	<?php endif ?>
+
+	<?php if ($system_logs_manage_log_form_hidden): ?>
+		<li>
+			<a onclick="$('#manage-log-form').toggle(360)" title="<?=gettext("Show/Hide manage log panel")?>">
+				<i class="btn-sm btn-breadcrumbbar">Manage Log</i>
+			</a>
```

Comment:
As a general rule there should be gettext() around anything that uses an "ordinary" word/s. That way the whole UI retains the potential to be translated into languages with completely different character sets. In "Latin character-based languages" we have some "IT words that are as good as international" and it feels silly to translate those, but other different script-bases need to be considered.
I think the exceptions to translation are some completely technical acronyms that have become so well-used in the IT industry that nobody would expand the acronym, translate it to another language, then make a new acronym out of it - "TCP "IP" "DHCP" "DNS" come to mind.
I am just having a brain-dump on the principle! I am currently doing a pass through the code to gettext() everything, so I am happy to do these after this PR is committed.